MTA-6811, 6812, 6813, 6814, 6815, 6816 CQA April Developer Lightspeed work (#357)
migtools/mta-documentation: docs/topics/developer-lightspeed/con_prerequisites.adoc

:_newdoc-version: 2.15.0
:_template-generated: 2024-2-21

:_mod-docs-content-type: CONCEPT

[id="prerequisites_{context}"]

> = Prerequisites

Reviewer: Please, remove this module completely.
 
Based on Ramon's feedback, I consolidated prereqs and persistent volume requirements: https://docs.redhat.com/en/documentation/migration_toolkit_for_applications/8.2/html/configuring_and_using_red_hat_developer_lightspeed_for_mta/getting-started_mta-developer-lightspeed#prerequisites_getting-started.

This must not be overwritten.

Author: Restored the new topic

= Prerequisites

[role="_abstract"]

> You can use the prerequisites required to successfully use the generative AI features in the {mta-dl-plugin} Visual Studio Code extension.

Reviewer: ```suggestion
Review the prerequisites required to use the generative AI features in the {mta-dl-plugin} Visual Studio Code extension.
```

> This section lists the prerequisites required to successfully use the generative AI features in the {mta-dl-plugin} Visual Studio Code extension.

Reviewer 2: This is self-referential and must be re-written. Perhaps something like this?

```suggestion
Review the prerequisites to successfully use the generative AI features in the {mta-dl-plugin} Visual Studio Code extension.
```

Fix if it occurs elsewhere too.
